previewer/metadata_renderer.js

```diff
@@ -45,6 +51,8 @@ setMetadataAssetID = function(ID) {
     )
 
     visibleProps.forEach(function(prop) {
+      if (!asset.hasOwnProperty(prop))
+        return;
```

Comment:
How were you envisioning that? I think we'd still have to have some list, because you would want the common properties to always appear in the same order.